README.md

```diff
@@ -27,21 +27,44 @@ plugins:[
     {
         use: 'gridsome-plugin-pwa',
         options: {
+            // Service Worker Options
+            disableServiceWorker: true,
+            serviceWorkerPath: 'service-worker.js',
+            cachedFileTypes: 'js,json,css,html,png,jpg,jpeg,svg,gif',
+
+            // Manifest Options (https://developer.mozilla.org/en-US/docs/Web/Manifest)
+            manifestPath: 'manifest.webmanifest',
             title: 'Gridsome',
+            shortName: 'Gridsome',              // Optional
+            description: 'Gridsome is awesome!',// Optional
```

Comment:
Default values have to be mentioned here. See heading above code block.
I suggest to have separate section for optional options. This is what I expect:
- Have a ready to go *minimal* configuration with defaults, so the user can just copy paste and use it.
- Have optional options in a different section so that user may use them when they need.

Reply:
I've gone with separating the options by type (service worker, manifest, meta tags, Microsoft Windows meta tags and App MacOS meta tags) which I think makes more sense.

Now moved all optional settings at the bottom of each section.